browser/net/brave_ad_block_tp_network_delegate_helper.h

```diff
@@ -16,6 +16,13 @@ int OnBeforeURLRequest_AdBlockTPPreWork(
     const ResponseCallback& next_callback,
     std::shared_ptr<BraveRequestInfo> ctx);
 
+int OnHeadersReceived_AdBlockCspWork(
```

Comment:
can we please move this to a separate file and also wrap with a `base::Feature flag`

Reply:
FWIW I was modeling this off of `ipfs_redirect_network_delegate_helper.h` which has both the `OnBeforeURLRequest` and `OnHeadersReceived` callbacks in the same file.